draft-ietf-scitt-architecture.md

```diff
@@ -366,287 +364,151 @@ Failure to produce this proof can indicate that the Transparency Services operat
 
-For a given Transparent Statement, Verifiers take as trusted inputs:
+Specific verifiable data structures, such those describes in {{-CT}} and {{-COMETRE}} are out of scope for this document.
 
-1. the CWT_Claims Issuer (or its resolved key manifest)
-1. the collection of Transparent Statements to which this Statement about the Artifact belongs (CWT_Claims Subject)
-1. the list of service identities of trusted Transparency Services
+### Adjacent Services
 
-When presented with a Transparent Statement for an Artifact, Verifiers verify the CWT_Claims Issuer identity, signature, and Receipt.
-They may additionally apply a validation policy based on the protected headers present both in the Envelope, the Receipt, or the Statement itself, which may include security-critical or Artifact-specific details.
+Transparency Services can be deployed along side other database or object storage technologies.
+For example, a Transparency Service that is supporting a software package management system, might be referenced from the APIs exposed for package management.
+Providing an ability to request a fresh receipt for a given software package, or to request a list of Signed Statements and Artifacts associated with a software package.
 
-Some Verifiers may systematically fetch all Transparent Statements using the CWT_Claims Subject and assess them alongside the Transparent Statement they are verifying to ensure freshness, completeness of evidence, and Non-equivocation.
+## Signed Statements
 
-Some Verifiers may choose to subset the collection of Statements, filtering on the payload type (Protected Header `3`), the CWT (Protected Header `15`) Issuer claim, or other non-opaque properties.
+This specification prioritizes conformance to {{RFC9052}} and its required and optional properties.
+Profiles and implementation specific choices should be used to determine admissability of conforming messages.
+This specification is left intentionally open to allow implementations to make the restrictions that make the most sense for their operational use cases.
 
-Some Verifiers may systematically resolve Issuer identifiers to fetch the latest corresponding verification keys.
-This behavior strictly enforces the revocation of compromised keys.
-Once the Issuer has updated its Statement to remove a key identifier, all Signed Statements include the corresponding `kid` will be rejected.
+At least one identifier for an identity document MUST be included in the protected header of the COSE envelope, either `x5t` or `kid`.
 
-Some Verifiers may decide to skip the identifier-based signature verification, relying on the Transparency Service's Registration Policy and the scrutiny of other Verifiers.
-Although this weakens their guarantees against key revocation, or against a corrupt Transparency Services, they can still keep the Receipt and blame the Issuer or the Transparency Services at a later point.
+- Support for `x5t` is mandatory to implement.
```

Comment:
That x5t is mandatory and kid is optional is very frustrating. It would be more helpful if was just one or the other must be present